web/yo/app/index.html

```diff
@@ -170,6 +170,8 @@
     <script src="components/angular-file-upload/angular-file-upload.min.js"></script>
     <script src="components/file-saver.js/FileSaver.js"></script>
     <script src="components/uuid-js/lib/uuid.js"></script>
+    <script src="components/google-diff-match-patch-js/diff_match_patch.js"></script>
+    <script src="components/google-diff-match-patch-js/diff_match_patch_uncompressed.js"></script>
```

Comment:
If I understand correctly from https://github.com/curran/google-diff-match-patch there is not need to include uncompressed version. If the bower config file for this project is actually including both of them, we should change the main file config in bower.json manually.